pytensor/graph/basic.py

```diff
@@ -193,11 +193,9 @@ def default_output(self):
             if len(self.outputs) == 1:
                 return self.outputs[0]
             else:
-                raise ValueError(f"{self.op}.default_output should be an output index.")
-        elif not isinstance(do, int):
-            raise ValueError(f"{self.op}.default_output should be an int or long")
-        elif do < 0 or do >= len(self.outputs):
-            raise ValueError(f"{self.op}.default_output is out of range.")
```

Comment:
I think Python error messages do a good enough job here when indexing is not valid or out of bounds. No need to reinvent the wheel.